app/javascript/src/PaymentGateways/braintree/BraintreeFormWrapper.jsx

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import * as React from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import { BraintreeForm } from 'PaymentGateways'
+import type { BraintreeFormProps } from 'PaymentGateways'
+
+const BraintreeFormWrapper = (props: BraintreeFormProps, containerId: string): void => (
+  createReactWrapper(<BraintreeForm { ...props } />, containerId)
+)
+
+export { BraintreeForm, BraintreeFormWrapper }
```

Comment:
`BraintreeForm` should be exported from `BraintreeForm.jsx`.
```suggestion
export { BraintreeFormWrapper }
```